CHANGELOG/v1.2.md

```diff
@@ -522,17 +527,29 @@ Other minor additions
   ⊎⇔∃      : (P zero ⊎ ∃⟨ P ∘ suc ⟩) ⇔ ∃⟨ P ⟩
   ```
 
+* Added new property to `Data.Fin.Subset`:
+  ```agda
+  _⊂_ : ∀ {n} → Subset n → Subset n → Set
+  _⊂′_ : ∀ {n} → Subset n → Subset n → Set
+  _⊄_ : ∀ {n} → Subset n → Subset n → Set
+  _⊄′_ : ∀ {n} → Subset n → Subset n → Set
+  ```
+
 * Added new proofs to `Data.Fin.Subset.Properties`:
   ```agda
   out⊆          : p ⊆ q → outside ∷ p ⊆      y ∷ q
   out⊆-⇔        : p ⊆ q ⇔ outside ∷ p ⊆      y ∷ q
   in⊆in         : p ⊆ q →  inside ∷ p ⊆ inside ∷ q
   in⊆in-⇔       : p ⊆ q ⇔  inside ∷ p ⊆ inside ∷ q
+  s⊆s           : p ⊆ q →       s ∷ p ⊆      s ∷ q
 
   ∃-Subset-zero : ∃⟨ P ⟩ → P []
   ∃-Subset-[]-⇔ : P [] ⇔ ∃⟨ P ⟩
   ∃-Subset-suc  : ∃⟨ P ⟩ → ∃⟨ P ∘ (inside ∷_) ⟩ ⊎ ∃⟨ P ∘ (outside ∷_) ⟩
   ∃-Subset-∷-⇔  : (∃⟨ P ∘ (inside ∷_) ⟩ ⊎ ∃⟨ P ∘ (outside ∷_) ⟩) ⇔ ∃⟨ P ⟩
+
+  x∉∁s⇒x∈s : ∀ {n} → {x : Fin n} → {s : Subset n} → x ∉ ∁ s → x ∈ s
+  x∉s⇒x∈∁s : ∀ {n} → {x : Fin n} → {s : Subset n} → x ∉ s → x ∈ ∁ s
```

Comment:
To keep things tidy, we usually don't put implicit variables in the types in the CHANGELOG

Reply:
I've removed all implicit variables from CHANGELOG.